package/MDAnalysis/units.py

```diff
@@ -104,13 +104,15 @@
 
      factor = 1 A**-3 / (N_Avogadro * (10**-9 dm)**-3)
 
-  relative to a density rho0 in g/cm^3::
+  relative to a density rho0 in :math:`g/cm^3`::
 
     M(H2O) = 18 g/mol   Molar mass of water
 
     factor = 1/(1e-24 * N_Avogadro / M(H2O))
+  
+  from :math:`rho/rho0 = n/(N_A * M^-1) / rho0`
```

Comment:
You could also type set the greek symbols here 
```suggestion
  from :math:`\rho/\rho_0 = n/(N_A * M^{-1}) / \rho_0`
```

Reply:
ok i will do the changes. one thing, can you tell which pep8 error you are talking about ? @PicoCentauri

Reply:
[See above](https://github.com/MDAnalysis/mdanalysis/pull/3634#issuecomment-1097726233)